master/usage/routereflector/bird-rr-config.md

```diff
@@ -44,18 +44,10 @@ single `bird` package installs both IPv4 and IPv6 BIRD):
 
 #### RHEL 7
 
-First, install EPEL. Depending on your system, the following command may
-be sufficient:
+Install BIRD. The BIRD packages can usually be found in the [EPEL](https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/b/ ) repository. We do not recommend adding EPEL as a base repository to your RHEL system to avoid packages in RHEL getting overwritten by EPEL versions.   
 
-    sudo yum install epel-release
-
-If that fails, try the following instead:
-
-    sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-
-With that complete, you can now install BIRD:
-
-    yum install -y bird{,6}
+    yum install https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/b/bird-1.4.5-2.el7.x86_64.rpm
+    yum install https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/b/bird6-1.4.5-2.el7.x86_64.rpm
```

Comment:
What happens when the version moves on from 1.4.5-2 ?

(Or is there good reason to believe that that will never happen?)

Reply:
Or we can suggest that folks install the latest bird/bird6 from https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/b/ without adding the EPEL repo.

Reply:
I think @kprabhak is bringing some real-wold RHEL expertise here: EPEL is considered a bit bleeding-edge in the RHEL world so most enterprises cherry-pick the odd package rather than adding the whole repo.

Reply:
I forget if there is a reason why the version of BIRD in RHEL7 is not good enough for RR purposes. @fasaxc do you remember?

If RHEL7 isn't good enough, another approach here would be to recommend the upstream BIRD team's PPA.

Reply:
To answer my own question there: there isn't any version of BIRD in base RHEL7, so that's why it isn't good enough :smile: (https://rpms.remirepo.net/rpmphp/zoom.php?rpm=bird).

@kprabhak What do you think about doc'ing use of the BIRD team's repo instead?  That would be per http://bird.network.cz/?download&tdir=redhat/

Reply:
@neiljerram I think that's a much better idea. Assuming of course that the bird repo stays up to date with rhel/centos releases, which it does appear to be.

Reply:
That does seem better, although it explicitly states that it's for Fedora, not RHEL.